setup.py

```diff
@@ -39,6 +37,8 @@
         version="0.1",
```

Comment:
PyPI requires uploading monotonically increasing versions, so we need to pay more attention here. What I did in CV is define __version__ in init.py and pull that here. You should consider doing something similar. Take a look at https://github.com/facebookresearch/ClassyVision/blob/master/setup.py

Reply:
Ah good call, I'll follow a similar pattern for CrypTen. I'm also a big fan of checking the Python version—will use that as well!